tools/topology/m4/eq_fir.m4

```diff
@@ -5,14 +5,13 @@ dnl Define macro for Eq effect widget
 dnl EQ name)
 define(`N_EQ_FIR', `EQFIR'PIPELINE_ID`.'$1)
 
-dnl W_EQ(name, format, periods_sink, periods_source, preload, kcontrols_list)
+dnl W_EQ(name, format, periods_sink, periods_source, kcontrols_list)
```

Comment:
@fredoh9 you removed preload, that means your $6 just became $5. can you please fix it?

Reply:
@ranj063 this is really good catch. That's was my bad. I tested with UP2 but not execute EQ pipeline. I will update this(and all comments) and test again before submitting the PRs.